tests/protocol/test_wire.py

```diff
@@ -1201,14 +1201,14 @@ def test_update_goal_state_should_save_goal_state(self):
 
         with open(extensions_config_file, "r") as file_:
             extensions_goal_state = ExtensionsGoalStateFactory.create_from_extensions_config(123, file_.read(), protocol)
-        self.assertEqual(4, len(extensions_goal_state.extensions), "Expected 4 extensions in the test ExtensionsConfig")
+        self.assertEqual(5, len(extensions_goal_state.extensions), "Expected 4 extensions in the test ExtensionsConfig")
```

Comment:
Looks like the expected value was updated, but not the message:
```python
"Expected 4 extensions in the test ExtensionsConfig"
```
instead of
```python
"Expected **5** extensions in the test ExtensionsConfig"
```